features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Apis/Details/ApiConsole/ApiConsole.jsx

```diff
@@ -518,26 +562,38 @@ class ApiConsole extends React.Component {
                                                 ),
                                             }}
                                         />
+                                        )}                                                                                         
                                     </Grid>
                                     <Grid x={12} md={6} className={classes.centerItems}>
-                                        {isApiKeyEnabled && (
+                                        {(isApiKeyEnabled || isBasicAuthEnabled || isOAuthEnabled) && (
                                             <FormControl component='fieldset' >
                                                 <RadioGroup
                                                     name='securityScheme'
                                                     value={securitySchemeType}
                                                     onChange={this.handleChanges}
                                                     row
                                                 >
-                                                    <FormControlLabel
-                                                        value='OAUTH'
+                                                    {isOAuthEnabled && (
+                                                      <FormControlLabel
+                                                      value='OAUTH'
+                                                      control={<Radio />}
+                                                      label='OAUTH'
+                                                  />
+                                                    )}
+                                                    {isBasicAuthEnabled && (
+                                                        <FormControlLabel
+                                                        value='BASIC'
                                                         control={<Radio />}
-                                                        label='OAUTH'
+                                                        label='BASIC'
```

Comment:
I think capitalizing these labels are wrong, it should be

- Basic
- Referenced ( OAuth )
- API Key
- Mutual SSL

Reply:
Done